presets/mainnet/eip-7732.yaml

```diff
@@ -0,0 +1,9 @@
+# Mainnet preset - EIP7732
+
+# Execution
+# ---------------------------------------------------------------
+# 2**9 (= 512)
+PTC_SIZE: 512
```

Comment:
In the tests, the number of initial validators is SLOTS_PER_EPOCH * 8. so, 32 * 8 = 256 in mainnet. The PTC_SIZE here is double than the amount of validators. This, combined with attestations by the PTC not counting, smells like potential finality issues in tests.

Reply:
ya, tests will fail for sure, we either decrease the ptc size for minimal or have to sample ptc like we do in sync committee, it's still an open question

Reply:
as in, why is the PTC even having an effect on finality weighing

Reply:
From the writing spec's perspective, it's easier to borrow PTC from the beacon attestation committee. 512 is a non-negotiable number, but as you said, this causes issues for minimal

Reply:
on mainnet preset, the tests run with 256 validators (32*8)